src/Event/Productions/DBALProductionRepository.php

```diff
@@ -77,6 +77,14 @@ public function removeEvent(string $eventId, ProductionId $productionId): void
         );
     }
 
+    /** @param string[] $eventIds */
+    public function removeEvents(array $eventIds, ProductionId $productionId): void
+    {
+        foreach ($eventIds as $eventId) {
+            $this->removeEvent($eventId, $productionId);
```

Comment:
Would it be safer to create a query builder and remove all the rows at once so they are wrapped in a transaction and multiple DB calls are avoided?